README.md

```diff
@@ -17,10 +17,20 @@ with the Python `wormhole` application.
 We need a version of [`haskell-magic-wormhole`][2] from git at the moment. This will
 go away as soon as a new release is uploaded into hackage.
 
-In order to build the code, we need Cabal 2.2 at the moment. A stack yaml file will
-be provided soon, for those who use `stack`.
+If you have cabal-install version < 2.2, then it is recommended to use cabal sandbox.
 
-The build steps using the cabal project (supported in cabal 2.x) is as follows:
+```
+cd wormhole-client
+cabal sandbox init
+cabal sandbox add-source /path/to/haskell-magic-wormhole/sources
+cabal install --only-dependencies --enable-tests
+cabal build
+```
```

Comment:
It is most likely because `happy` is not installed on your setup. One thing I have consistently found is that a lot of things (most notably `cabal-install` itself build-depends on `alex` and `happy`, so I almost always install them by default via `cabal-install`. The `-v` flag to `cabal` command can also show some additional information, which is sometimes helpful.

I will try creating a fresh VM and do these steps and add additional dependencies to be installed into the `README`. Thanks.